doc/source/management/audit_log_plugin.rst

```diff
@@ -63,19 +63,19 @@ Installation
 
 Audit Log plugin is shipped with |Percona Server|, but it is not installed by default. To enable the plugin you must run the following command: 
 
-.. code-block:: mysql
+.. code-block:: guess
 
    INSTALL PLUGIN audit_log SONAME 'audit_log.so';
 
```

Comment:
This currently results in https://jira.percona.com/browse/PS-4853. I'm not sure if this feature docs need to be added if the feature can't be installed.

Reply:
I'd say let's leave it in